deepfence_ui/app/scripts/components/vulnerability-view/cve-image-report-container.js

```diff
@@ -14,46 +16,65 @@ class CVEImageReportContainer extends React.PureComponent {
     this.handleDownload = this.handleDownload.bind(this);
   }
 
-  handleDownload(params) {
-    const {
-      xlsxReportDownloadAction: action,
-    } = this.props;
+  async handleDownload(params) {
+    const { dispatch } = this.props;
```

Comment:
why take dispatch in props? when we can use `useDispatch`?

Reply:
These files are still class based. They are part of the list which needs to be refactored. Once this fix is pushed, I'll refactor them to functional components.